docs/api.md

```diff
@@ -462,16 +462,20 @@ Forces the maximum disk space to be used by the disk cache. Value is given in by
 --inject <value>
 ```
 
-Allows you to inject a javascript or css file. This command can be run multiple times to inject the files.
+Allows you to inject a javascript or css file. This command can be run multiple times to inject the files. This can also be a valid raw Github Gist Url.
```

Comment:
In addition to updating the MD doc, please update the `--help` help in cli.ts:

https://github.com/nativefier/nativefier/blob/380c98b23d0ac288e2aaefa844d98d6710e9bcc4/src/cli.ts#L216